examples/use_cases/creating_reusable_components/reusing_previous_work.ipynb

```diff
@@ -0,0 +1,902 @@
+       "      <td>491</td>\n",
+       "      <td>-1.507202</td>\n",
+       "      <td>-1.377293</td>\n",
+       "      <td>-0.265119</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>183.598370</td>\n",
+       "      <td>0.135285</td>\n",
+       "      <td>2</td>\n",
+       "      <td>-0.000055</td>\n",
+       "      <td>323</td>\n",
+       "      <td>51615</td>\n",
+       "      <td>541</td>\n",
+       "      <td>-0.195758</td>\n",
+       "      <td>-0.028410</td>\n",
+       "      <td>-0.155695</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>183.680207</td>\n",
+       "      <td>0.126185</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0.123111</td>\n",
+       "      <td>287</td>\n",
+       "      <td>52023</td>\n",
+       "      <td>513</td>\n",
+       "      <td>1.297604</td>\n",
+       "      <td>-0.590023</td>\n",
+       "      <td>0.140338</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>183.870529</td>\n",
+       "      <td>0.049911</td>\n",
+       "      <td>2</td>\n",
+       "      <td>-0.000111</td>\n",
+       "      <td>3306</td>\n",
+       "      <td>54922</td>\n",
+       "      <td>510</td>\n",
+       "      <td>-1.446117</td>\n",
+       "      <td>0.566685</td>\n",
+       "      <td>-0.009272</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>183.883288</td>\n",
+       "      <td>0.102557</td>\n",
+       "      <td>2</td>\n",
+       "      <td>0.000590</td>\n",
+       "      <td>3306</td>\n",
+       "      <td>54922</td>\n",
+       "      <td>512</td>\n",
+       "      <td>-0.849271</td>\n",
+       "      <td>1.287505</td>\n",
+       "      <td>-0.397689</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "           ra       dec  class  redshift  plate    mjd  fiberid     PCA_1  \\\n",
+       "0  183.531326  0.089693      2 -0.000009   3306  54922      491 -1.507202   \n",
+       "1  183.598370  0.135285      2 -0.000055    323  51615      541 -0.195758   \n",
+       "2  183.680207  0.126185      0  0.123111    287  52023      513  1.297604   \n",
+       "3  183.870529  0.049911      2 -0.000111   3306  54922      510 -1.446117   \n",
+       "4  183.883288  0.102557      2  0.000590   3306  54922      512 -0.849271   \n",
+       "\n",
+       "      PCA_2     PCA_3  \n",
+       "0 -1.377293 -0.265119  \n",
+       "1 -0.028410 -0.155695  \n",
+       "2 -0.590023  0.140338  \n",
+       "3  0.566685 -0.009272  \n",
+       "4  1.287505 -0.397689  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sdss_df_fe = sdss_df\n",
+    "\n",
+    "# encode class labels to integers\n",
```

Comment:
split this encoding and do it before "u, g, r, i, z" section. I'd argue its not quite "Feature Engineering" even but im ok with it being in "Feature Engineering" section under something like "Encoding labels"